applications/asset_tracker_v2/src/ext_sensors/Kconfig

```diff
@@ -9,6 +9,13 @@ menuconfig EXTERNAL_SENSORS
 
 if EXTERNAL_SENSORS
 
+config EXTERNAL_SENSORS_ACTIVITY_DETECTION_AUTO
+	bool "Set accelerometer trigger handler on initialization"
```

Comment:
I think the option description should match the new option name. `Start activity detection automatically` perhaps. Users can go to the help text for more explanation